tracer/src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -413,6 +413,10 @@ _ when x.ToBoolean() is { } boolean => boolean,
                                    .AsBoolResult()
                                    .OverrideWith(in otelRuntimeMetricsEnabled, ErrorLog, defaultValue: false);
 
+            DataPipelineEnabled = config
+                                  .WithKeys(ConfigurationKeys.TraceDataPipelineEnabled)
+                                  .AsBool(defaultValue: true);
```

Comment:
Just a reminder to set this false before the final merge 🙂 
```suggestion
                                  .AsBool(defaultValue: false);
```

Reply:
https://github.com/DataDog/dd-trace-dotnet/pull/6925

Reply:
> So are we or are we not changing the default to false (for now)?

We removed it from that PR, so we should reinstated it here, right @ganeshnj?